src/main/java/hudson/plugins/throttleconcurrents/ThrottleJobProperty.java

```diff
@@ -199,10 +203,14 @@ public boolean isThrottleMatrixConfigurations() {
         return categoryTasks;
     }
     private static Item getItem(ItemGroup group, String name) {
-        if (group instanceof Jenkins) {
-            return ((Jenkins) group).getItemMap().get(name);
-        } else {
+        SecurityContext orig = SecurityContextHolder.getContext();
+        NotSerilizableSecurityContext auth = new NotSerilizableSecurityContext();
+        auth.setAuthentication(ACL.SYSTEM);
+        SecurityContextHolder.setContext(auth);
```

Comment:
`ACL#impersonate` is not available in Jenkins 1.424, which is targeted by throttle-concurrent-build.

|Method                                          |Introduced version|
|:-----------------------------------------------|:-----------------|
|`ACL#impersonate(Authentication)`               |1.462             |
|`ACL#impersonate(Authentication, Runnable)`     |1.509             |
|`ACL#impersonate(Authentication, Callable<V,T>)`|1.587             |